Documentation/btrfs-subvolume.rst

```diff
@@ -252,7 +252,7 @@ show [options] <path>
         -u|--uuid UUID
                 show details about subvolume with the given *UUID*, looked up in *path*
 
-snapshot [-r] [-i <qgroupid>] <source> <dest>|[<dest>/]<name>
+snapshot [-r|-R] [-i <qgroupid>] <source> <dest>|[<dest>/]<name>
```

Comment:
Please also add a long option, in this case it follows `ls` so -R and --recursive.

Reply:
Done

Reply:
I guess David also means the long option should also show up in the doc like:

`snapshjot [-r|-R|--recursive] [-i <qgroupid>] <source> <dest>|[<dest>/]<name`

Reply:
Oh, okay, have done. I was following what other commands do in this case, but it looks like it's inconsistent.